src/main/java/core/basesyntax/animals/Animal.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.animals;
+
+public abstract class Animal {
```

Comment:
let's add some state to our animal (represented by fields) so we will use inheritance feature

Reply:
I hope I understood correctly what you meant

Reply:
Yep, that's what i meant